frontend/src/app/interceptors/error/error.interceptor.ts

```diff
@@ -12,16 +13,23 @@ export class ErrorInterceptor implements HttpInterceptor {
 
   intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
     return next.handle(request).pipe(catchError(err => {
-
       if (err.status === 401) {
         // auto logout if 401 response returned from api
         this._authService.logout();
-        throw new Error('Špatné přihlašovací údaje');
+        throw new Error('Uživatel nenalezen. Zkontrolujte prosím správnost přihlašovacích údajů.');
```

Comment:
pridal bych jeste `403` - pristup byl odmitnut, pokud si myslite, ze sem mate pristup kontaktujte administratora na ockovani@mild.blue  - a k tmu znovu to requestid